src/signals/incident-management/containers/IncidentDetail/index.js

```diff
@@ -214,6 +238,8 @@ const IncidentDetail = () => {
     dispatch({ type: CLOSE_ALL });
   }, []);
 
+  if (!Object.keys(handlingTimesBySlug).length) return null;
```

Comment:
Is this check necessary?

Reply:
With this in place the page will load after handling times are loaded (like it happens on a page refresh)

Reply:
```JavaScript
return Object.keys(handlingTimesBySlug).length && state.incident ? (
  <IncidentDetailContext.Provider>
    ...
  </IncidentDetailContext.Provider>
) : null;
```

Reply:
Don't you want to show a loading indicator instead?

Reply:
With the designer agreed that this slows down showing the page on slower connections so it shoulsd be removed. A new solution that is proposed in another story will solve that issue.